documentation/modules/operators/con-operator-topic-names.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-operator-topic-names-{context}']
+= Topic naming conventions
+
+A `KafkaTopic` resource includes a name for the topic and a label that identifies the name of the Kafka cluster it belongs to. 
+
+.Label identifying a Kafka cluster for topic handling
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: topic-name-1
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  topicName: topic-name-1  
+----
+
+The cluster name is derived from the `Kafka` resource.
+If the label does not match the Kafka cluster, the Topic Operator cannot identify the `KafkaTopic`, and the topic is not created.
```

Comment:
The cluster label is IMHO used when the TO is deployed using the KAfka resource. But Standalone TO might not use it. Might be worth mentioning?

Reply:
Right. I've added a note:

> NOTE: The label is only used when deploying the Topic Operator with the `Kafka` resource.
> When deploying the standalone Topic Operator, you specify a connection to a Kafka cluster instead.